api/models/Goal.js

```diff
@@ -8,24 +8,9 @@
 module.exports = {
 
   attributes: {
-    name: {
-      type: 'string'
-    }
+	  name:{
```

Comment:
You removed the old Goal.js, there was a comment in it. So you should add unique status field integer type in order query faster.